src/libraries/System.Linq/tests/AnyTests.cs

```diff
@@ -63,6 +63,19 @@ public void Any(IEnumerable<int> source, bool expected)
             Assert.Equal(expected, source.Any());
         }
 
+        [Fact]
+        public void Any_Grouping()
+        {
+            var grouppedValues = new int[] { 1, 2 }.GroupBy(num => num);
+            Assert.True(grouppedValues.Any());
+
+            var grouppedValuesToEmpty = string.Empty.GroupBy(ch => ch);
+            Assert.False(grouppedValuesToEmpty.Any());
+
+            var anoymousGroupValues = "MmmmalllleDDDonggg".GroupBy(ch => ch, (k, v) => v);
```

Comment:
We generally try to avoid incorporating personal identifiable information in test data.
```suggestion
            var anoymousGroupValues = "AaaaBbbbCcccddd".GroupBy(ch => ch, (k, v) => v);
```